### PR TITLE
Chore: spelling - theme

### PR DESCRIPTION
**Pull request by jsoref**
**What this PR does / why we need it**:

This fixes a number of spelling errors.

**Special notes for your reviewer**:

Comments on file changes can be found in #24412

### CODE REVIEW

**Comment by jsoref** [comment]
@sakjur ?

File: public/sass/components/_panel_graph.scss [hunk 1]
```diff
@@ -321,15 +321,15 @@
   top: 50%;
   left: 8px;
   transform: translateX(-50%) translateY(-50%) rotate(-90deg);
-  // this is needed for phantomsjs 2.1
+  // this is needed for phantomjs 2.1
   -webkit-transform: translateX(-50%) translateY(-50%) rotate(-90deg);
```
Comment (dprokop) [inline public/sass/components/_panel_graph.scss:325]: we can remove these lines

File: public/sass/components/_panel_graph.scss [hunk 2]
```diff
@@ -321,15 +321,15 @@
...
 }
 
 .right-yaxis-label {
   top: 50%;
   right: 8px;
   transform: translateX(50%) translateY(-50%) rotate(90deg);
-  // this is needed for phantomsjs 2.1
+  // this is needed for phantomjs 2.1
   -webkit-transform: translateX(50%) translateY(-50%) rotate(90deg);
```
Comment (dprokop) [inline public/sass/components/_panel_graph.scss:333]: we can remove these lines

**Review by dprokop — CHANGES_REQUESTED** [review]
Thanks for your contribution @jsoref! Can you please remove the lines I have commented on and we should be good to go?

**Review by dprokop — APPROVED** [review]
Now it looks good

**Pull request merged**